src/main/java/br/usjt/apivolei/maestro/model/service/ExperienciaService.java

```diff
@@ -110,8 +110,27 @@ public boolean adquirir(Experiencia experiencia, Torcedor torcedor) {
 			return false;
 		}
 	}
+	
+	public boolean cancelar(Experiencia experiencia, Torcedor torcedor) {
+		boolean torcedorRemovido = experiencia.removeTorcedor(torcedor);
+		
+		if(torcedorRemovido) {
+			experiencia.setQtdDisponivel(experiencia.getQtdDisponivel() + 1);
+			
+			IPonto pontoTorcedor = new PontoTorcedor(torcedor.getPontos().doubleValue());
+			pontoTorcedor.incrementar(experiencia.getCusto());
```

Comment:
- Neste ponto deve ser setado o valor da nova puntação para o torcedor relacionado

```suggestion
			torcedor.setPontos(pontoTorcedor.incrementar(experiencia.getCusto()));
```